Dockerfile.nginx

```diff
@@ -0,0 +1,31 @@
+
+FROM ezsystems/php:7.1-v1 as app
+
+ENV SYMFONY_ENV=prod
+
+# Copy in project files into work dir
+COPY . /var/www
+
+# assetic:dump requires parameters.yml, so let's generate it
+RUN composer run-script build --no-interaction
```

Comment:
You have now added a doc/docker/Dockerfile-nginx file.
That obsoletes this file entirely and I'll remove it